### PR TITLE
CCS Mop Up Changes Part 2

### PR DESCRIPTION
**Pull request by berroar**
### What is the context of this PR?

Content mop up updates for CSS.

### How to review

Check the changes match the google [doc](https://docs.google.com/spreadsheets/d/1JZDyX0RS2nkZj5A4kXvxubdGr_BV2Hk6XGnSRq6AClA/edit#gid=879728302)

### Checklist

- [ ] Jsonnet files conform to the latest [style guide](/ONSdigital/eq-questionnaire-schemas/blob/master/style_guide.md)

#### England

- [CCS](https://test-launcher.gcp.dev.eq.ons.digital/quick-launch?language_code=en&survey=CENSUS&form_type=I&region_code=GB-ENG&url=https://storage.googleapis.com/eq-questionnaire-schemas-artifacts/ccs-mop-up-2/schemas/en/ccs_household_gb_eng.json)

### CODE REVIEW

**Comment by berroar** [comment]
[(I) CCS- Mop up changes 2 (XS)](https://trello.com/c/nCmbOPdP/3995-i-ccs-mop-up-changes-2-xs)

File: source/jsonnet/england-wales/ccs/blocks/who-lives-here/outside_uk_note.jsonnet [hunk 1]
```diff
@@ -7,7 +7,7 @@ local placeholders = import '../../../lib/placeholders.libsonnet';
   content: {
     title: 'End of interview',
     instruction: {
-      text: 'If the respondent was living outside the UK on Sunday {census_date}, please explain that they do not need to complete the questionnaire and end interview',
+      text: 'If the respondent was living outside the UK on Sunday {census_date}, please explain that they do not need to complete the questionnaire and end interview <p>You still need to submit this questionnaire.</p><p> Continue past the household members summary page and select Submit</p>',
```
Comment (LJBabbage) [inline source/jsonnet/england-wales/ccs/blocks/who-lives-here/outside_uk_note.jsonnet:10]: The dreaded full stop missing *2

File: source/jsonnet/england-wales/ccs/blocks/who-lives-here/primary_person_list_collector.jsonnet [hunk 2]
```diff
@@ -56,13 +56,13 @@ local placeholders = import '../../../lib/placeholders.libsonnet';
...
           {
-            description: 'The address at which you generally spend most time, for most people this will be their permanent or family home.',
+            description: 'The address at which you generally spend most time. For most people this will be their permanent or family home.',
           },
           {
             list: [
-              '<strong>full-time students</strong> should include themselves here',
-              '<strong>armed forces members</strong> should include themselves at their home address if they have one',
-              '<strong>include anyone temporarily outside the UK, if this was for less than 12 months</strong>',
+              '<strong>Full-time students</strong> should include themselves here.',
```
Comment (LJBabbage) [inline source/jsonnet/england-wales/ccs/blocks/who-lives-here/primary_person_list_collector.jsonnet:63]: Although this is correct according to the spreadsheet, I am under the impression that list items start with small letters

File: source/jsonnet/england-wales/ccs/blocks/who-lives-here/who_lives_here_interstitial.jsonnet [hunk 3]
```diff
@@ -15,13 +15,13 @@ local placeholders = import '../../../lib/placeholders.libsonnet';
         },
       },
       {
-        description: 'A <em>household</em> is one person living alone, or a group of people (not necessarily related), who share cooking facilities, and share a\n        living room, <em>or</em> sitting room, <em>or</em> dining area.',
+        description: 'A <em>household</em> is one person living alone, or a group of people (not necessarily related), who share cooking facilities <em>and</em> share a\n        living room, <em>or</em> sitting room, <em>or</em> dining area.',
```
Comment (LJBabbage) [inline source/jsonnet/england-wales/ccs/blocks/who-lives-here/who_lives_here_interstitial.jsonnet:18]: Shouldn't this be strong, not em?

Comment (berroar) [reply source/jsonnet/england-wales/ccs/blocks/who-lives-here/who_lives_here_interstitial.jsonnet]: I used em cause it was already there in the rest of the sentence, but have updated them all to be strong now.

File: source/jsonnet/england-wales/ccs/blocks/who-lives-here/primary_person_list_collector.jsonnet (hunk 2 above)
Comment (berroar) [reply source/jsonnet/england-wales/ccs/blocks/who-lives-here/primary_person_list_collector.jsonnet:63]: I will double check with @donnap573 on this

**Comment by berroar** [comment]
> Just the odd thing, I also couldn't see changes for Interviewer note (line 5, 7, 20, 28) or line 15 and 17. I have a feeling another card might have already done this, but as I checked the PR against the spreadsheet and not the html I can't tell.

@LJBabbage BTW, the interviewer note change was done in [this PR](https://github.com/ONSdigital/eq-questionnaire-runner/pull/202)

**Pull request merged**